src/sentry/models/avatar.py

```diff
@@ -0,0 +1,95 @@
+from __future__ import absolute_import
+
+from django.db import models, transaction
+from PIL import Image
+from six import BytesIO
+from uuid import uuid4
+
+from sentry.db.models import FlexibleForeignKey, Model
+from sentry.utils.cache import cache
+
+
+class AvatarBase(Model):
+    """
+    Base class for UserAvatar and OrganizationAvatar models, which
+    associate users/orgs with their avatar preferences/files.
+    """
+    __core__ = False
+
+    ALLOWED_SIZES = (20, 32, 48, 52, 64, 80, 96, 120)
```

Comment:
this needs to be updated to `ALLOWED_SIZES = (20, 32, 36, 48, 52, 64, 80, 96, 120)` i think. (i think i added 36 as an allowed size to fix a UI bug since initially doing this model)